server/v2/testdata/app.toml

```diff
@@ -1,3 +1,21 @@
+[comet]
```

Comment:
Main server doesn't contain store as it is its own go.mod. Is it only used to test viper? If not that isn't needed

Reply:
only for testing

Reply:
Okay but what test uses it?